### PR TITLE
renaming components for consistency w/ current product vision and naming rhetoric

### PR DESCRIPTION
**Pull request by ShariqAli-Dev**
# Description
 - [Loom Video](https://www.loom.com/share/2302cc965dd44d60b8a2c36dcbfc8cbd)
 - The current codebase has outdated naming rhetoric with odd component names. To prevent conflicts and confusion in the future, I have renamed components and written notes on components that were too complex to simply rename (which was only 1 component).
 - For instance, clients were referred to as households and even family sometimes. And since most people asked clarifying questions about the same component rhetoric, might as well cement it early on while there aren't many components.

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Change status
- [x] Complete, but not tested (may need new tests)

# How Has This Been Tested?
 - Since I haven't implemented any new features and simply renamed components,  this PR had manual testing. I look at routes, rendered all components changed and all worked well without any bugs. Future pull requests will have at minimum basic unit testing applied.

# Checklist:
- [x] I have pulled from master before submitting this PR
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer (the peer review to approve a PR counts.  The reviewer must download and test the code)
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by ShariqAli-Dev** [comment]
[BUGFIX: Renaming components in frontend for consistently with backend & current product vision](https://trello.com/c/r4dw3O1o/46-bugfix-renaming-components-in-frontend-for-consistently-with-backend-current-product-vision)

**Review by stkirk — APPROVED** [review]
Looks good Shariq. The "client" nomenclature is more clear than "family" and will help future devs on this project know intuitively what data is being referenced

**Comment by wild-kard** [comment]
Some much needed changes to the component verbiage. This will make it a lot clearer for future dev teams to parse what these components are supposed to do. 

Good catch on including the changes to the routes in this as well.

**Review by wild-kard — APPROVED** [review]
Some much needed changes to the component verbiage. This will make it a lot clearer for future dev teams to parse what these components are supposed to do.

Good catch on including the changes to the routes in this as well.

File: src/components/common/ClientIntakeForm/index.js [hunk 1]
```diff
@@ -127,5 +133,3 @@ export const FamilyIntake = () => {
     </div>
   );
 };
```
Comment (chavionjackson) [inline src/components/common/ClientIntakeForm/index.js:135]: Nice job changing this. This should help avoid confusion

**Review by chavionjackson — COMMENTED** [review]
Naming rhetoric will be a lot more clear

File: src/components/common/ClientIntakeForm/index.js (hunk 1 above)
Comment (ShariqAli-Dev) [reply src/components/common/ClientIntakeForm/index.js:135]: Thank you!

**Pull request merged**